Core/clim-core/commands/parsers.lisp

```diff
@@ -247,8 +252,10 @@
                           ,seperator
                           ,(keyword-arg-name-from-symbol arg)
                           ,seperator)
-                  (present ,key-arg-val ,ptype-form
-                           :stream ,stream))
+                  (if (unsupplied-argument-p ,key-arg-val)
+                      (with-text-face (stream :italic)
```

Comment:
Missing unquote for `stream`? Only relevant if no function is defined.

Reply:
fixed.